src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -78,7 +78,7 @@
 
 		<para>There are multiple ways to add a new job to DataCleaner monitor.
 			The
-			most fail-safe way is to use one of the job wizards. These are
+			safest way is to use one of the job wizards. These are
```

Comment:
I don't really like any of the, but I like the old one best: To me, _fail-safe_ implies that you can't screw it up (it's safe against failure) while _safest_ implies that it is making a dangerous operation safer.

... But I think I'd prefer "easiest".